<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>4</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>MPR</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="MultiPlanarReformatWidget" name="mprView" native="true">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>5</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Volume</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="VolumeProjectionWidget" name="volumeView" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenFile"/>
    <addaction name="actionOpenDirectory"/>
    <addaction name="separator"/>
    <addaction name="actionOpenProject"/>
    <addaction name="actionSaveProject"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuStereo">
    <property name="title">
     <string>Stereo</string>
    </property>
    <widget class="QMenu" name="menuStereoMode">
     <property name="title">
      <string>Stereo Mode</string>
     </property>
     <addaction name="actionStereoAnaglyph"/>
     <addaction name="actionStereoInterlaced"/>
     <addaction name="actionStereoOff"/>
    </widget>
    <addaction name="actionStereoDistance"/>
    <addaction name="actionStereoEyeAngle"/>
    <addaction name="menuStereoMode"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionSeriesSelector"/>
    <addaction name="actionSegmentationTools"/>
    <addaction name="separator"/>
    <addaction name="actionCubicInterpolation"/>
   </widget>
   <widget class="QMenu" name="menuSeries">
    <property name="title">
     <string>Series</string>
    </property>
    <addaction name="actionLoadAllSeries"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSeries"/>
   <addaction name="menuView"/>
   <addaction name="menuStereo"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockSeriesSelector">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>5</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0" colspan="2">
      <widget class="QTreeView" name="treeView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
       <property name="uniformRowHeights">
        <bool>true</bool>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockSegmentation">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="0" column="0">
      <widget class="QFrame" name="frame">
       <layout class="QGridLayout" name="gridLayout_5">
        <property name="margin">
         <number>0</number>
        </property>
        <item row="1" column="0" colspan="2">
         <widget class="QFrame" name="frame2">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="lineWidth">
           <number>0</number>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Draw Size:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBoxSize">
             <property name="maximum">
              <number>50</number>
             </property>
             <property name="value">
              <number>5</number>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>5</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QSlider" name="sliderSize">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>50</number>
          </property>
          <property name="value">
           <number>5</number>
          </property>
          <property name="sliderPosition">
           <number>5</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::NoTicks</enum>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QPushButton" name="buttonDilate">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Dilate</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QPushButton" name="buttonErode">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Erode</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QCheckBox" name="checkErase">
          <property name="text">
           <string>erase</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="buttonDraw">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Draw</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="buttonThreshold">
          <property name="text">
           <string>Threshold</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="buttonRegionGrow">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Region Grow</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0" colspan="2">
         <widget class="QPushButton" name="buttonAnalyse">
          <property name="text">
           <string>Analyse</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="statusTip">
    <string>Info about this Application</string>
   </property>
  </action>
  <action name="actionOpenFile">
   <property name="text">
    <string>Open &amp;File(s)</string>
   </property>
   <property name="statusTip">
    <string>Open a file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="statusTip">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionStereoDistance">
   <property name="text">
    <string>Distance</string>
   </property>
  </action>
  <action name="actionStereoEyeAngle">
   <property name="text">
    <string>Eye Angle</string>
   </property>
  </action>
  <action name="actionStereoAnaglyph">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Anaglyph</string>
   </property>
  </action>
  <action name="actionStereoInterlaced">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Interlaced</string>
   </property>
  </action>
  <action name="actionStereoOff">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Off</string>
   </property>
  </action>
  <action name="actionOpenDirectory">
   <property name="text">
    <string>Open &amp;Directory</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionSeriesSelector">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>SeriesSelector</string>
   </property>
  </action>
  <action name="actionNewSegmentation">
   <property name="text">
    <string>new Segmentation</string>
   </property>
   <property name="toolTip">
    <string>create a new Segmentation</string>
   </property>
  </action>
  <action name="actionSegmentationTools">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Segmentation Tools</string>
   </property>
  </action>
  <action name="actionLoadAllSeries">
   <property name="text">
    <string>Load All</string>
   </property>
  </action>
  <action name="actionCubicInterpolation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Cubic Interpolation</string>
   </property>
  </action>
  <action name="actionSaveProject">
   <property name="text">
    <string>Save Project</string>
   </property>
  </action>
  <action name="actionOpenProject">
   <property name="text">
    <string>Open Project</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MultiPlanarReformatWidget</class>
   <extends></extends>
   <header>multiplanarreformatwidget.h</header>
  </customwidget>
  <customwidget>
   <class>VolumeProjectionWidget</class>
   <extends></extends>
   <header>volumeprojectionwidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>treeView</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>actionSeriesSelector</sender>
   <signal>toggled(bool)</signal>
   <receiver>dockSeriesSelector</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>101</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dockSeriesSelector</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>actionSeriesSelector</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>101</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSegmentationTools</sender>
   <signal>toggled(bool)</signal>
   <receiver>dockSegmentation</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>136</x>
     <y>494</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dockSegmentation</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>actionSegmentationTools</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>494</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
